pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	balanceResourcePriority = &ResourceAllocationPriority{"BalanceResourceAllocation", balancedResourceScorer}
```

Comment:
now that you have fixed this, could you please rename the variable to `balancedResourcePriority`?

Reply:
@bsalamat Done.